core/templates/dev/head/components/version_diff_visualization_directive.html

```diff
@@ -32,7 +32,8 @@ <h4>Exploration Graph Legend</h4>
     pointer-events: none;
   }
   .legend-graph {
-    position: absolute;
+    left: 100px;
```

Comment:
I'm not sure why we need left 100px as we already have right -100px? (If it's required then maybe you can remove right -100px?)

Reply:
hi @dubeysandeep I have left: 100px take this beacause i shift the EXPLORATION GRAPH LEGEND but i think that's will be not needed. so I change this.